controller/controller_manager.go

```diff
@@ -67,8 +67,8 @@ func StartControllers(logger logrus.FieldLogger, stopCh chan struct{}, controlle
 	//  lead to scalability problems, since we dump the whole cache of each object back in to the reconciler every 30 seconds.
 	//  if a specific controller requires a periodic resync, one enable it only for that informer, add a resync to the event handler, go routine, etc.
 	//  some refs to look at: https://github.com/kubernetes-sigs/controller-runtime/issues/521
-	kubeInformerFactory := informers.NewSharedInformerFactory(kubeClient, time.Second*30)
-	lhInformerFactory := lhinformers.NewSharedInformerFactory(lhClient, time.Second*30)
+	kubeInformerFactory := informers.NewSharedInformerFactory(kubeClient, time.Hour)
+	lhInformerFactory := lhinformers.NewSharedInformerFactory(lhClient, time.Hour)
```

Comment:
I seem to remember backup related setting also rely on this resync to ensure the settings take effect. Not sure if this change will impact that, or it's not a case anymore?

Reply:
No. Right now there will be a separate goroutine periodically update `backupTarget.spec.syncRequestedAt` then trigger backup & backupVolume CR update. But let's wait for Jenting's response as confirmation.
@jenting Can you help verify this won't affect backup-related scenarios like BackupVolume creation?

Reply:
I remember no, the backup-related scenarios would not depend on the resync time.
As Shuo's comment, the backup-related depends on the `spec.syncRequestedAt` field.

Reply:
https://github.com/longhorn/longhorn-manager/blob/840ae441597ac6af0904946db7d40916d959f9f8/webhook/client/client.go#L46-L47

Please check if we need to change the resync time of the webhook server as well.
cc @derekbit

Reply:
Is there any potential risk after increasing resync period to 1 hour?

Reply:
@derekbit For example, if we increase this interval to 1 hour for the node controller, then Longhorn cannot detect the disk failure (via checking the disk meta file in the reconcile loop) in time. That's why there is an exception. 

Here, we want to verify if there are more cases like the node controller.

Reply:
@jenting I think the resync time of the webhook server can also increase to 1 hour, because it only reads resources such as settings from the datastore.

Reply:
I'd like no resync if possible.

> But, I'm not sure if there are other parts in webhooks requiring short resync period, so I will re-check the resyncPeriod after finishing mutating and validating webhooks.

Thank you.